lib/manager/components/version/FeedVersionSpanChart.js

```diff
@@ -2,137 +2,270 @@
 
 import moment from 'moment'
 import React, { Component } from 'react'
-import { Table } from 'react-bootstrap'
+import { OverlayTrigger, Tooltip } from 'react-bootstrap'
 
+import VersionDateLabel from './VersionDateLabel'
 import type { FeedVersion } from '../../../types'
 
 type Props = {
   activeVersion: FeedVersion,
-  comparedVersion: FeedVersion,
-  // Used for display only.
-  comparedVersionIndex: number
+  comparedVersion?: ?FeedVersion,
+  style: any
 }
 
-// Type used for rendering FeedSpan sub-components.
-type VersionDateRange = {
-  // Number of days for this range
-  days: number,
-  endDate: moment$Moment,
-  // Length and offset should be ratios.
-  length: number,
-  offset: number,
-  startDate: moment$Moment
-}
+const DATE_FORMAT = 'MMM. D, YYYY'
```

Comment:
No dot needed. Otherwise this can be weird for May.

![image](https://user-images.githubusercontent.com/2370911/117718372-fe5c8e00-b1a9-11eb-87fd-4478a6ce8330.png)